tests/functional/aws-node-sdk/test/multipleBackend/objectCopy.js

```diff
@@ -261,7 +278,97 @@ describeSkipIfNotMultiple('MultipleBackend object copy', function testSuite() {
                     assert.strictEqual(result.CopyObjectResult.ETag,
                         `"${correctMD5}"`);
                     assertGetObjects(key, bucket, awsLocation, copyKey, bucket,
-                        awsLocation, copyKey, 'COPY', false, done);
+                        awsLocation, copyKey, 'COPY', false, awsS3,
+                        awsLocation, done);
+                });
+            });
+        });
+
+        it('should copy an object on AWS to a different AWS location ' +
+        'with source object READ access',
+        done => {
+            const awsConfig2 = getRealAwsConfig(awsLocation2);
+            const awsS3Two = new AWS.S3(awsConfig2);
+            const copyKey = `copyKey-${Date.now()}`;
+            const awsBucket =
+                config.locationConstraints[awsLocation].details.bucketName;
+            async.waterfall([
+                next => awsS3.putBucketAcl({
+                    Bucket: awsBucket,
+                    ACL: 'public-read',
+                }, err => next(err)),
+                // giving access to the object on the AWS side
+                next => putSourceObj(awsLocation, false, key =>
+                  next(null, key)),
+                (key, next) => awsS3.putObjectAcl(
+                  { Bucket: awsBucket, Key: key,
+                  ACL: 'public-read' }, err => next(err, key)),
+                (key, next) => {
+                    const copyParams = {
+                        Bucket: bucket,
+                        Key: copyKey,
+                        CopySource: `/${bucket}/${key}`,
+                        MetadataDirective: 'REPLACE',
+                        Metadata: {
+                            'scal-location-constraint': awsLocation2 },
+                    };
+                    process.stdout.write('Copying object\n');
+                    s3.copyObject(copyParams, (err, result) => {
+                        assert.equal(err, null, 'Expected success ' +
+                        `but got error: ${err}`);
+                        assert.strictEqual(result.CopyObjectResult.ETag,
+                            `"${correctMD5}"`);
+                        next(err, key);
+                    });
+                },
+                (key, next) =>
+                assertGetObjects(key, bucket, awsLocation, copyKey,
+                  bucket, awsLocation2, copyKey, 'REPLACE', false,
+                  awsS3Two, awsLocation2, next),
+            ], done);
+        });
+
+        it('should return error AccessDenied copying an object on AWS to a ' +
```

Comment:
As discussed, please add a note in the docs that one account can't copy to another account with a source and destination on real AWS unless the account associated with the access Key/secret Key pairs used for the source bucket has rights to put in the destination bucket. ACL's would have to be updated on AWS directly to enable this.